src/components/Footer/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import Img from '../../components/Img';
+
+import EdxLogo from '../../images/edx-logo.png';
+import './Footer.scss';
+
+class Footer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      enterpriseLogoNotFound: false,
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    const { enterpriseLogo } = this.props;
+    if (enterpriseLogo && enterpriseLogo !== prevProps.enterpriseLogo) {
+      this.setState({ // eslint-disable-line react/no-did-update-set-state
+        enterpriseLogoNotFound: false,
+      });
+    }
+  }
+
+  renderEnterpriseLogo() {
+    const { enterpriseLogo, enterpriseSlug, enterpriseName } = this.props;
+    return (
+      <Link className="logo" to={`/${enterpriseSlug}`}>
+        <Img
+          src={enterpriseLogo}
+          alt={`${enterpriseName} logo`}
+          onError={() => this.setState({ enterpriseLogoNotFound: true })}
```

Comment:
The `onerror` attribute on an `img` handles the case when the browser tries to load an image, but the image fails to load (e.g., the image no longer exists at that URL, network issues, etc.).

Here, when the image can't load, we don't want to show any image at all. I handle this by setting state that the image can't be loaded which then prevents the `renderEnterpriseLogo` function was being called (see the logic in the `render` function on line 50).